fhir-search/src/main/java/com/ibm/fhir/search/SearchConstants.java

```diff
@@ -95,7 +98,7 @@ private SearchConstants() {
 
     // set as unmodifiable
     public static final List<String> SEARCH_RESULT_PARAMETER_NAMES =
-            Collections.unmodifiableList(Arrays.asList(SORT, COUNT, PAGE, INCLUDE, REVINCLUDE, ELEMENTS, SUMMARY));
+            Collections.unmodifiableList(Arrays.asList(SORT, COUNT, PAGE, INCLUDE, REVINCLUDE, ELEMENTS, SUMMARY, TOTAL));
```

Comment:
Is the order important? Perhaps use a Set<String> instead?

Reply:
Order doesn't matter, so I can change it to a Set.